src/OmniSharp.Abstractions/Models/v2/BlockStructureRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿using OmniSharp.Mef;
+
+namespace OmniSharp.Models.v2
+{
+    [OmniSharpEndpoint(OmniSharpEndpoints.V2.BlockStructure, typeof(BlockStructureRequest), typeof(BlockStructureResponse))]
+    public class BlockStructureRequest : Request
```

Comment:
This should probably just inherit from `SimpleFileRequest`. It doesn't need to support all of the properties from `Request`.